src/installer.py

```diff
@@ -69,26 +76,23 @@ def dlstatus():
                 
                 pyotherside.send('whatState',"=> "+ str(size) +"/687180204 bytes (" + str(round(size / 687180204 * 100, 2)) + "%)")
                 
-                if size >= 680000000:
-                    pyotherside.send('whatState',"=> Almost done!")
+                if size == 687180200:
+                    pyotherside.send('whatState',"=> 5 Minutes Left!")
                     print("Download status stops now")
                     run = False
                 else: pass
             
-
         trd1 = threading.Thread(target=download)
         trd2 = threading.Thread(target=dlstatus)
         
         trd1.start()
         trd2.start()
 
-        #reboot
-        child.expect('root.*', timeout=10000)
-        print("reboot")
-        pyotherside.send('whatState',"=> rebooting")
-        child.sendline("reboot")
-        child.expect("root.*", timeout=180)
-        time.sleep(1000)
+        #wait for the threads to finish
+        trd1.join()
+        trd2.join()
```

Comment:
And because you `join` the download thread, the `child.expect` with a timeout will never happen, since the thread will never exit.

Reply:
But it does......... I get to the final message fine xd